src/main/java/com/fakng/fakngagrgtr/parser/apple/responseBody.json

```diff
@@ -0,0 +1,3423 @@
+{
+  "cmsPacketData": {
```

Comment:
Can you please make it shorter? It's enough to leave only one vacancy example from the entire file. We don't need the whole page there. It's just JSON example

Besides, naming should be rather apple-sample.json